mlocal/frags/Makefile.stub

```diff
@@ -79,6 +81,12 @@ $(starter_suid_INSTALL): $(starter)
 	$(V)install -d $(@D)
 	$(V)install -m 4755 $(starter) $(starter_suid_INSTALL)
 
+# Run-Singularity bin script
+$(dist_bin_SCRIPTS_INSTALL): $(dist_bin_SCRIPTS)
+	@echo " INSTALL" $@
```

Comment:
Change this to:
`@echo " INSTALL run-singularity"`

Reply:
yeah this is correct.